src/api-service/__app__/onefuzzlib/proxy.py

```diff
@@ -258,6 +285,20 @@ def get_or_create(cls, region: Region) -> Optional["Proxy"]:
                     proxy.state = VmState.stopping
                     proxy.save()
                 return None
+            if proxy.timestamp is not None:
+                proxy_timestamp = proxy.timestamp
+                if proxy_timestamp < (datetime.datetime.now() - datetime.timedelta(7)):
```

Comment:
Something is wrong with the line. I tried to deploy an instance and when I run the debug command, it errors out on this line and eventually causes a 500.